### PR TITLE
Hipp 1254 del prod credentials

### PR DESCRIPTION
**Pull request by sedathad**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala [hunk 1]
```diff
@@ -64,8 +67,8 @@ class CredentialsSteps @Inject()(sharedState: SharedState) extends BaseStepDef {
   }
 
   Then("""there is/are {int} production credential(s)""") { (expectedCount: Int) =>
-    EnvironmentAndCredentialsPage(sharedState.application.id)
-      .viewProductionEnvironment()
+    EnvironmentAndCredentialsPage(sharedState.application.id, Some(ProductionTab))
+//      .viewProductionEnvironment()
```
Comment (PaulCDurham) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala:71]: Please remove commented out code

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala [hunk 2]
```diff
@@ -81,4 +84,38 @@ class CredentialsSteps @Inject()(sharedState: SharedState) extends BaseStepDef {
...
+    ProductionCredentialsSuccessPage(sharedState.application.id).returnToEnvironmentsAndCredentials()
+
+    EnvironmentAndCredentialsPage(sharedState.application.id)
+      .viewProductionEnvironment()
+
+  }
+
+  When("""the user revokes the first production credential""") { () =>
+    EnvironmentAndCredentialsPage(sharedState.application.id, Some(ProductionTab))
+//      .revokeFirstTestCredential()
```
Comment (PaulCDurham) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala:117]: Please remove commented out code

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala [hunk 3]
```diff
@@ -81,4 +84,38 @@ class CredentialsSteps @Inject()(sharedState: SharedState) extends BaseStepDef {
     EnvironmentAndCredentialsPage(sharedState.application.id, Some(TestTab)).hasTestCredential(clientId)
   }
 
+  Then("""the recently created production credential still exists""") { () =>
+    EnvironmentAndCredentialsPage(sharedState.application.id, Some(ProductionTab)).hasProductionCredential(clientId)
+  }
+
+  When("""the user creates TWO production credentials""") { () =>
```
Comment (PaulCDurham) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala:91]: There is redundant and duplicated code in here.

Add this to `Journeys.scala`:

```
  def createProductionCredential(sharedState: SharedState): EnvironmentAndCredentialsPage = {
    EnvironmentAndCredentialsPage(sharedState.application.id)
      .viewProductionEnvironment()
      .createNewProductionCredential()
      .confirmAndContinue()
      .returnToEnvironmentsAndCredentials()
  }
```

Then have this in the step def:

```
    Journeys
      .createProductionCredential(sharedState)
    Journeys
      .createProductionCredential(sharedState)
      .viewProductionEnvironment()

```

**Review by PaulCDurham — CHANGES_REQUESTED** [review]

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala (hunk 1 above)
Comment (sedathad) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala:71]: Done

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala (hunk 2 above)
Comment (sedathad) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala:117]: Done

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala (hunk 3 above)
Comment (sedathad) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CredentialsSteps.scala:91]: Updated.

**Pull request merged**